ext/reflection/tests/ReflectionGenerator_basic.phpt

```diff
@@ -37,7 +37,7 @@ foreach ($gens as $gen) {
 --EXPECTF--
 object(Generator)#2 (0) {
 }
-array(1) {
+array(2) {
```

Comment:
I'm not sure about these test changes. Based on previous results it looks like this was supposed to be the "inner" trace of the generator. Maybe @bwoebi can comment.

Reply:
The existing PHP code misses the top frame of generators, that is also informative. See the following array(0) -> array(1) changes.
@bwoebi was this intended?

Reply:
actually, I'm not sure if the existing backtrace format is good. Simple reflection of VM stack might be better. It would show file/line position of the last ``yield``.

Reply:
While @nikic is correct, yes, it was supposed to be that way, I think including the top frame of the generator as well is a good idea.

I also don't think this will break anything as usages of ReflectionGenerator I've seen in the wild are just formatting the trace for debug output.